src/index.vue

```diff
@@ -22,7 +30,8 @@ export default {
         ...this.barStyle,
         'animation-duration': this.duration,
         'animation-direction': this.direction,
-        'animation-delay': this.delay
+        'animation-delay': this.delay,
+        'animation-play-state' : (this.paused) ? "paused" : "running"
```

Comment:
please use single-quote `'` instead of a double-quote `"`.

Reply:
Done